widget/tabcontainer_test.go

```diff
@@ -56,3 +57,52 @@ func TestTabContainer_ApplyTheme(t *testing.T) {
 	Renderer(tabs).ApplyTheme()
 	assert.NotEqual(t, barColor, underline.FillColor)
 }
+
+func TestTabContainerRenderer_Layout(t *testing.T) {
+	tabs := NewTabContainer(
+		NewTabItemWithIcon("Text1", theme.CancelIcon(), canvas.NewCircle(theme.BackgroundColor())),
+		NewTabItemWithIcon("Text2", theme.ConfirmIcon(), canvas.NewCircle(theme.BackgroundColor())),
+	)
+
+	r := Renderer(tabs)
+	r.Layout(fyne.NewSize(100, 100))
+
+	images, texts := collectImagesAndTexts(r.Objects(), fyne.NewPos(0, 0))
+
+	if assert.Len(t, images, 2) && assert.Len(t, texts, 2) {
+		img1 := images[0]
+		img2 := images[1]
+		if img1.Position().X > img2.Position().X {
+			img2, img1 = img1, img2
+		}
+		text1 := texts[0]
+		text2 := texts[1]
+		if text1.Position().X > text2.Position().X {
+			text2, text1 = text1, text2
+		}
+
+		assert.GreaterOrEqual(t, text1.Position().X, img1.Position().X+img1.Size().Width)
```

Comment:
GreaterOrEqual seems like a warning sign - should we not be testing for Equal and getting the value right? (I am guessing it's missing a + theme.Padding())?

Reply:
Okay, checking for a fixed distance seems reasonable.

Reply:
cool